mine/models/stg_customers.sql

```diff
@@ -0,0 +1,15 @@
+with (
+    select * 
+    from {{source('ecommerce', 'customers')}}
+    ) as source 
```

Comment:
stylistic point: we alias in front of CTEs as though they were functions (e.g. `def x():` in python). We find this makes for more readable code. See the examples here: https://app.getguru.com/card/cM54ddki/Ordering-Columns-in-dbt

Reply:
done :)